docs-conceptual/azps-6.1.0/install-az-ps.md

```diff
@@ -49,6 +50,18 @@ platforms. Run the following command from a PowerShell session:
 ```powershell-interactive
 Install-Module -Name Az -Scope CurrentUser -Repository PSGallery -Force
 ```
+Then you need to import the module:
+
+```powershell-interactive
+Import-Module az.accounts
+```
```

Comment:
Explicitly importing the Az.Accounts module isn't required. Module autoloading functionality was added to PowerShell in version 3.0.